src/nstack.c

```diff
@@ -358,8 +358,12 @@ int main(int argc, char *argv[])
     if (handle == -1) {
         perror("Failed to init");
         exit(1);
+    } else if (handle == -2) {
+        perror("Interface identifier is too long");
+        exit(1);
     }
 
+
```

Comment:
Eliminate this blank line to minimize the changes.

Reply:
Blank line removed. I did the changes on the GitHub web interface hence 4 commits, you may want to squash them.